root/relationship/linkattributetype/RelationshipAttributeTypeIndex.js

```diff
@@ -0,0 +1,197 @@
+                href={'/relationship-attribute/' + attribute.gid + '/delete'}
+              >
+                {l('Remove')}
+              </a>
+            )}
+          </span>
+        ) : null}
+
+        {attribute.description ? (
+          <>
+            <h2>{l('Description')}</h2>
+            <p>{expand2react(l_relationships(attribute.description))}</p>
+          </>
+        ) : null}
+
+        <p>
+          {$c.user?.is_relationship_editor ? (
+            <>
+              <strong>{l('ID:')}</strong>
+              {' '}
+              {attribute.id}
+              <br />
+              <strong>{l('Child order:')}</strong>
+              {' '}
+              {attribute.child_order}
+              <br />
+            </>
+          ) : null}
+          <strong>{l('UUID:')}</strong>
+          {' '}
+          {attribute.gid}
+          <br />
+          {parent ? (
+            <>
+              <strong>{addColonText(l('Parent attribute'))}</strong>
+              {' '}
+              <a href={'/relationship-attribute/' + parent.gid}>
+                {upperFirst(l_relationships(parent.name))}
+              </a>
+            </>
+          ) : null}
+        </p>
+
+        {childrenAttrs.length ? (
+          <>
+            <h2>{l('Possible values')}</h2>
+            <ul>
+              {childrenAttrs
+                .slice(0)
+                .sort((a, b) => compareChildren(a, b))
+                .map(attribute => (
+                  <AttributeTree
+                    attribute={attribute}
+                    key={attribute.gid}
+                  />
+                ))}
+            </ul>
+          </>
+        ) : null}
+
+        <h2>{l('Relationship usage')}</h2>
+        {relationships.length ? (
+          <>
+            <p>
+              {l(`This relationship type is being used
+                  by the following relationships:`)}
+            </p>
+
+            {relationships.map(relationship => (
+              <React.Fragment key={relationship.gid}>
+                <h3>
+                  <a
+                    href={'/relationship/' + relationship.gid}
+                  >
+                    {upperFirst(l_relationships(relationship.name))}
+                  </a>
+                  {' '}
+                  {bracketedText(texp.l(
+                    '{type0} - {type1}',
+                    {
+                      type0: formatEntityTypeName(relationship.type0),
+                      type1: formatEntityTypeName(relationship.type1),
+                    },
+                  ))}
+                </h3>
+                {relationship.description ? (
+                  <p>
+                    {expand2react(l_relationships(relationship.description))}
+                  </p>
+                ) : null}
+              </React.Fragment>
+            ))}
+          </>
+        ) : (
+          <p>
+            {l(`This relationship type isn’t directly being used
+                by any relationships.`)}
```

Comment:
```suggestion
            {l(`This attribute isn’t directly being used
                by any relationship types.`)}
```
BTW, not sure how relevant this message is on child attribute types, since link type attributes can *only* refer to root attribute types. But it probably doesn't matter too much.